File: app/src/main/java/com/dewonderstruck/apps/ashx0/ui/blog/list/BlogListActivity.java
package com.dewonderstruck.apps.ashx0.ui.blog.list;

import android.os.Bundle;


import com.dewonderstruck.apps.ashx0.R;

import com.dewonderstruck.apps.ashx0.ui.common.PSAppCompactActivity;
import com.dewonderstruck.apps.ashx0.databinding.ActivityBlogListBinding;

import androidx.databinding.DataBindingUtil;

public class BlogListActivity extends PSAppCompactActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        ActivityBlogListBinding binding = DataBindingUtil.setContentView(this, R.layout.activity_blog_list);

        initUI(binding);

    }

    private void initUI(ActivityBlogListBinding binding) {

        // Toolbar
        initToolbar(binding.toolbar, getResources().getString(R.string.blog_list__title));

        // setup Fragment

        setupFragment(new BlogListFragment());

    }
}

File: app/src/main/java/com/dewonderstruck/apps/ashx0/ui/blog/list/BlogListActivity.kt
package com.dewonderstruck.apps.ashx0.ui.blog.list

import android.os.Bundle
import androidx.databinding.DataBindingUtil
import com.dewonderstruck.apps.ashx0.R
import com.dewonderstruck.apps.ashx0.databinding.ActivityBlogListBinding
import com.dewonderstruck.apps.ashx0.ui.common.PSAppCompactActivity
class BlogListActivity : PSAppCompactActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val binding: ActivityBlogListBinding = DataBindingUtil.setContentView(this, R.layout.activity_blog_list)
        initUI(binding)
    }

    private fun initUI(binding: ActivityBlogListBinding) {

        // Toolbar
        initToolbar(binding.toolbar, resources.getString(R.string.blog_list__title))

        // setup Fragment
        setupFragment(BlogListFragment())
    }
}